column/api/objects/__init__.py

```diff
@@ -0,0 +1,3 @@
+class State(object):
+    RUNNING = "RUNNING"
```

Comment:
im ok with that but we do need to be able to access the results as in my comment below